src/api/event.ts

```diff
@@ -15,6 +20,27 @@ import { waitForIndexerOnVersion } from "./utils";
 export class Event {
   constructor(readonly config: AptosConfig) {}
 
+  /**
+   * Get module events by event type
+   *
+   * @param args.eventType - The event type
+   * @param args.minimumLedgerVersion Optional ledger version to sync up to, before querying
+   *
+   * @returns Promise<GetEventsResponse>
+   */
+  async getModuleEventsByEventType(args: {
+    eventType: MoveStructId;
```

Comment:
Can you add a check here that if the event type is too long (wrapped for example) we should send a warning and possibly error. We cap indexed_event at length 300.

Reply:
Added to [internal function](https://github.com/aptos-labs/aptos-ts-sdk/pull/304/files#diff-e2ce900e3281d047225ea1fd842abcb5fa55614f33438d27e1b1b6e342127ccbR101)